src/app/pages/vm/vm-wizard/vm-wizard.component.ts

```diff
@@ -156,18 +156,20 @@ export class VMWizardComponent {
           isHidden: false
         },
         {
-          type: 'select',
+          type: 'explorer',
           name: 'datastore',
-          placeholder : T('Select a datastore'),
-          tooltip: T('Choose a datastore for the new zvol.'),
+          placeholder : T('Select a zpool/datastore'),
```

Comment:
As noted in the other bug, "datastore" is a VMware term. In ZFS, they are datasets.  Additionally, "zpool" is redundant with just "pool".

So this should be just T('Select a pool or datastore')